docker-compose.yml

```
@@ -1,11 +1,11 @@
-services:
-  mediaflow-proxy:
+services:  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)